### ISSUE TITLE
IDE-R not working on Linux

### ISSUE THREAD

**Issue by jacobopantoja** [Ylianst/MeshCommander#23]
When I run Meshcommander (currently 0.8.6a, installed through npm) on Linux, IDE-R does not work properly.

I can select an ISO, it is displayed for some seconds (0 in, 18 out), and then disappears.

If I run MeshCommander in a Windows machine, installed with the official installer, IDE-R works properly.

The target machine is the same in both cases.

**Comment by Ylianst** [comment]
Current version of MeshCommander on NPM is 0.8.8-a. Can you confirm that is the version you have?

**Comment by jacobopantoja** [comment]
Hi Ylian, it seems that the problem persists in 0.8.8-a. I can remember I did some extra testing but I cannot find the information nor I did post it; I'll either find it or re-do the testing to shed some extra light on this topic, this evening.

**Comment by jacobopantoja** [comment]
I've done a bit of testing. In 0.8.6a, IDE-R disconnects after (0 in, 18 out) being displayed for a while.

In 0.8.8a (both NPM and msi invoked with SHIFT when launching IDE-R) it is properly working.

I've tested in an 8.1 ME firmware machine and an 11.6 one. In both I get approximately equal results, although the 8.1 one is a bit buggy and thus I misinterpreted that it was not working, while the problem was that the BIOS itself was not launching the IDE-R devices.

So, somewhere between 0.8.6 and 0.8.8 (perhaps 060f012d910cf74d7dfcf90897395894593790b1?) this bug got solved, therefore I'm closing
